src/main/java/amazin/User.java

```diff
@@ -0,0 +1,85 @@
+package amazin;
+
+import javax.persistence.*;
+
+@Entity
+public class User {
+
+    @Id
+    @GeneratedValue(strategy= GenerationType.AUTO)
+    @Column(name = "id", updatable = false, nullable = false)
+    private Long id = null;
+    private String firstName;
+    private String lastName;
+    private String email;
+    private String password;
+
+    public User(){}
+
+    public User(Long id, String firstName, String lastName, String email, String password) {
+        this.id = id;
+        this.firstName = firstName;
+        this.lastName = lastName;
+        this.email = email;
+        this.password = password;
+    }
+
+
+    public Long getId() {
+        return id;
+    }
+
+    public void setId(Long id) {
```

Comment:
int

Reply:
same As previous comment, long is the standard for ids.